CRAMv3.tex

```diff
@@ -1378,19 +1378,30 @@ \subsubsection*{Read feature codes}
 
 \subsubsection*{Base substitution codes (BS data series)}
 
-A base substitution is defined as a change from one nucleotide base (reference 
-base) to another (read base) including N as an unknown or missing base. There are 
-5 possible bases ACGTN, 4 possible substitutions for each base and 20 substitutions 
-in total. Substitutions for the same reference base are assigned integer codes 
-from 0 to 3 inclusive. To restore a base one would need to know its substitution 
-code and the reference base. 
+A base substitution is defined as a change from one nucleotide base (reference base) to
+another (read base), including N as an unknown or missing base. There are 5 possible reference
+bases (ACGTN), with 4 possible substitutions for each base, and 20 substitutions in total.
+The codes for all possible substitutions are stored in a substitution matrix. To restore a
+base, one would use the reference base and the substitution code, resolving the base via lookup
+in the substitution matrix.
 
-A base substitution matrix assigns integer codes to all possible substitutions. 
+\subsubsection*{Substitution Matrix Format}
 
-Substitution matrix is written as follows. Substitutions for a given reference 
-base are sorted by their frequencies in descending order then assigned numbers 
-from 0 to 3. Same-frequency ties are broken using alphabetical order. For example, 
-let us assume the following substitution frequencies for base A: 
+Each of the 4 possible substitutions for a given reference base is assigned a 2-bit integer
+code (see below) with a value ranging from 0 to 3 inclusive. The 4 2-bit codes are packed
+into a single byte,  in the fixed order ACGTN, and the entire substitution matrix is
```

Comment:
This is a little confusing as it could be implied that our 2-bit codes are packed together in the order ACGTN (although that would be 5 2-bit codes and not 4).  it doesn't particularly spell out that this is ACGTN minus the specific reference base being encoded. 

"Each of the 4 possible substitutions for a given reference base is assigned a 2-bit integer code (see below) with a value ranging from 0 to 3 inclusive. The 4 2-bit codes are packed into a single byte, high 2-bits first, for each base ACGTN (minus the reference base itself).   The entire substitution matrix is written as 5 such bytes, one for each reference base, also in the order ACGTN."

Reply:
Yeah, done.